backend/cmd/server/main.go

```diff
@@ -73,8 +70,8 @@ func main() {
 		logger.Fatal("Failed to load private key", log.Error(err))
 	}
 
-	// Register the services.
-	registerServices(mux, jwtService)
+	// Register the services and get observability service for shutdown
+	observabilitySvc := registerServices(mux, jwtService)
```

Comment:
Rather than getting it back to the main.go just to shutdown. Can't we have an `unRegisterServices()` function in the servicemanager.go and call it from the main.go before calling the `gracefulShutdown`

In servicemanager.go, we can keep the observabilitySvc outside and Shutdown